code/script.js

```diff
@@ -2,8 +2,11 @@
 // We haven't really talked about ´this´ yet, but we will... ;)
 // console.log(this) to see how it works
 function toggle() {
-  this.classList.toggle("active")
+    this.classList.toggle("active");
 }
 
 // Selects an HTML element, and calls a function which will be executed when the element is clicked.
-document.getElementById("section1").onclick = toggle
+document.getElementById("section1").onclick = toggle;
+document.getElementById("section2").onclick = toggle;
+document.getElementById("section3").onclick = toggle;
+document.getElementById("section4").onclick = toggle;
```

Comment:
Great job, but we could done it in some loop to make it universal, what do you think?

Reply:
I really did not have a clue what I was doing here with js, but it worked. Hopefully I will learn better ways to solve these things the weeks to come.

Reply:
I see, crossing fingers for that! For future if you don't understand any concept, please consider asking questions on Stack Overflow!